packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/observability/probe/NewConnectionActivatedProbeTest.kt

```diff
@@ -68,7 +68,8 @@ internal abstract class NewConnectionActivatedProbeTest(
                     event.properties[TelemetryProperty.IS_LOCALHOST] == isLocalhost &&
                     event.properties[TelemetryProperty.IS_ENTERPRISE] == isEnterprise &&
                     event.properties[TelemetryProperty.IS_GENUINE] == isGenuine &&
-                    event.properties[TelemetryProperty.VERSION] == version
+                    event.properties[TelemetryProperty.VERSION] == version &&
+                    event.properties[TelemetryProperty.ATLAS_HOST] == ""
```

Comment:
What does this test? Why `event.properties[TelemetryProperty.ATLAS_HOST] == ""` expects always empty string? Shouldn't it be `atlastHost` like e.g. `version`.

Reply:
This is an integration test that connects to a local cluster in Atlas and to a local cluster created with the Atlas CLI and checks the different inferred properties, that's why the ATLAS_HOST is always empty.

Reply:
Got it! Then only the same question as previously if we could make it null instead of empty string to match telemetry across all tools.